docs/9.0/api/sidebar.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
```

Comment:
Can you update the code of this file? I assume that it should be the same as https://github.com/handsontable/handsontable/pull/8684/files#diff-3fed73e5abb4ad2f1ac16483eeda6322844db460e54b08a95a0c5cff84205541R1-R27?

Reply:
No, it shouldn't. The API references for 9.0 were generated in a different way than later.